### PR TITLE
docs(i18n): include translation status in contributing guide

### PR DESCRIPTION
**Pull request by userquin**
This PR also updates nuxt-i18n link: old one pointing to Nuxt 2 docs

### CODE REVIEW

**Review by Shinigami92 — APPROVED** [review]
🤯 😻

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -89,7 +89,14 @@ We've added some `UnoCSS` utilities styles to help you with that:
 
 ## Internationalization
 
-We are using [vue-i18n](https://vue-i18n.intlify.dev/) via [nuxt-i18n](https://i18n.nuxtjs.org/) to handle internationalization.
+We are using [vue-i18n](https://vue-i18n.intlify.dev/) via [nuxt-i18n](https://v8.i18n.nuxtjs.org/) to handle internationalization.
+
+You can check current [translation status](https://elk-docs.netlify.app/docs/guide/contributing#translation-status): more instructions on table caption.
```
Comment (edimitchel) [inline CONTRIBUTING.md:94]: Isn't `docs.elk.zone` ?

```suggestion
You can check current [translation status](https://docs/elk.zone/docs/guide/contributing#translation-status): more instructions on table caption.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -89,7 +89,14 @@ We've added some `UnoCSS` utilities styles to help you with that:
...
+
+If you are updating a translation in your local environment, you can run the following commands to check the status:
+- from root folder: `nr prepare-translation-status`
+- change to `docs` folder and run docs dev server `cd docs && nr dev`
```
Comment (edimitchel) [inline CONTRIBUTING.md:98]:
```suggestion
- change to `docs` folder and run docs dev server `nr dev`
```
change to docs folder is redundant

File: CONTRIBUTING.md (hunk 1 above)
Comment (userquin) [reply CONTRIBUTING.md:94]: it is `docs.elk.zone`

Comment (userquin) [reply CONTRIBUTING.md:94]: cannot apply your suggestion, the link is wrong ;)

Comment (edimitchel) [reply CONTRIBUTING.md:94]: updated

**Review by edimitchel — APPROVED** [review]

**Pull request merged**